manual/doc/extending.tex

```diff
@@ -88,4 +88,94 @@ \section{How to add a new observable}
+    \item[\texttt{options}] the map represenation of the options pertaining to the observable;
+    \item[\texttt{physical-limit}] the physical lower bound on the observable;
+    \item[\texttt{theta}] the $\theta$ parameter of the Amoroso distribution;
+    \item[\texttt{alpha}] the $\alpha$ parameter of the Amoroso distribution;
+    \item[\texttt{beta}] the $\beta$ parameter of the Amoroso distribution;
+\end{description}
+We strongly recommend contacting the \EOS authors before adding your own constraint of type \texttt{Amoroso}.
+
+\subsection{Type \texttt{Gaussian}}
+
+Type \texttt{Gaussian} uses a univariate Gaussian or normally-distributed
+likelihood. The following keys are required in the description of the
+constraint:
+\begin{description}[labelwidth=.15\textwidth]
+    \item[\texttt{observable}] the qualified name of an \texttt{Observable} or a \texttt{Parameter};
+    \item[\texttt{kinematics}] the map representation of the kinematic variables pertaining to the observable;
+    \item[\texttt{options}] the map represenation of the options pertaining to the observable;
+    \item[\texttt{mean}] the mean of the distribution;
+    \item[\texttt{sigma-stat}] the map representation with keys \texttt{hi} and \texttt{lo} for the
+    upper and lower statistical uncertainty of the distribution, respectively;
+    \item[\texttt{sigma-sys}] the same as \texttt{sigma-stat}, but for the systematic uncertainty;
+    \item[\texttt{dof}] the degrees of freedom associated with this observation (should default to $1$).
+\end{description}
+
+\subsection{Type \texttt{MultivariateGaussian}}
+
+Type \texttt{MultivariateGaussian} uses a multivariate Gaussian likelihood,
+parametrised in terms of its mode and correlation matrix. The following keys
+are required in the description of the constraint:
+\begin{description}[labelwidth=.15\textwidth]
+    \item[\texttt{dim}] the dimensionality of the multivariate distribution;
+    \item[\texttt{observable}] the list of size \texttt{dim} of qualified names for for either \texttt{Observable}s or \texttt{Parameter}s;
+    \item[\texttt{kinematics}] the list of size \texttt{dim} of map representations of the kinematic variables pertaining to each observable;
+    \item[\texttt{options}] the list of size \texttt{dim} of map represenations of the options pertaining to each observable;
+    \item[\texttt{mean}] the list of size \texttt{dim} describing the mean of the distribution;
+    \item[\texttt{sigma-stat-hi}] the list of size \texttt{dim} for the upper statistical uncertainty of the distribution;
+    \item[\texttt{sigma-stat-lo}] the list of size \texttt{dim} for the lower statistical uncertainty of the distribution;
+    \item[\texttt{correlations}] the matrix of size \texttt{dim} $\times$ \texttt{dim} for the statistical correlations;
+    \item[\texttt{sigma-sys}] the list of size \texttt{dim} for the systematic uncertainty;
```

Comment:
Describe how 'sigma-sys' is combined with the other 'sigma-stat-hi', 'sigma-stat-lo' and 'correlations'.

Reply:
Done.